### ISSUE TITLE
Cannot build labelflow (webpack errors)

### ISSUE THREAD

**Issue by mihav** [labelflow/labelflow#460]
**Describe the bug**
Build fails on `yarn build:web` step because of webpack errors. 

**To Reproduce**
Clone labelflow repo and perform `yarn install` and `yarn build:web`.

**Additional context**
 - Tested on Ubuntu 18.04 or 20.04
 - nodejs upgraded to 16.10.0 from official repo
 - yarn upgraded to 1.22.5 from official repo

**Output Log**
```shell
export SENTRY_LOG_LEVEL=debug
vagrant@ubuntu2004:~/labelflow$ yarn build:web
Prisma schema loaded from schema.prisma

✔ Generated Prisma Client (2.30.3) to ./../../node_modules/@prisma/client in 424ms
You can now start using Prisma Client in your code. Reference: https://pris.ly/d/client

import { PrismaClient } from '@prisma/client'
const prisma = new PrismaClient()

info  - Loaded env from /home/vagrant/labelflow/typescript/web/.env.local
info  - Using webpack 5. Reason: Enabled by default https://nextjs.org/docs/messages/webpack5
warn  - You have enabled experimental feature(s).
warn  - Experimental features are not covered by semver, and may cause unexpected or broken application behavior. Use them at your own risk.

info  - Checking validity of types  
warn  - The Next.js plugin was not detected in your ESLint configuration. See https://nextjs.org/docs/basic-features/eslint#migrating-existing-config

./src/pages/api/auth/[...nextauth].ts
60:7  Warning: Unexpected console statement.  no-console
64:7  Warning: Unexpected console statement.  no-console
67:7  Warning: Unexpected console statement.  no-console

./src/components/class-selection-popover/__stories__/class-selection-popover.tsx
64:3  Warning: Unexpected alert.  no-alert
77:32  Warning: Unexpected console statement.  no-console
94:32  Warning: Unexpected console statement.  no-console

./src/components/datasets/__stories__/dataset-card.tsx
23:22  Warning: Unexpected console statement.  no-console
24:24  Warning: Unexpected console statement.  no-console

./src/components/labelling-tool/drawing-tool-bar/__stories__/drawing-tool.tsx
44:33  Warning: Unexpected console statement.  no-console
46:28  Warning: Unexpected console statement.  no-console

./src/components/labelling-tool/options-tool-bar/class-selection-menu/__stories__/class-selection-menu.tsx
71:3  Warning: Unexpected alert.  no-alert

./src/components/website/Navbar/NavContent.tsx
59:34  Warning: Do not use Array index in keys  react/no-array-index-key
61:34  Warning: Do not use Array index in keys  react/no-array-index-key
97:29  Warning: Do not use Array index in keys  react/no-array-index-key

./src/components/website/Navbar/Submenu.tsx
37:22  Warning: Do not use Array index in keys  react/no-array-index-key
75:34  Warning: Do not use Array index in keys  react/no-array-index-key

./src/components/website/Pricing/PricingCard.tsx
60:46  Warning: Do not use Array index in keys  react/no-array-index-key

./src/components/website/Pricing/PricingTable.tsx
47:54  Warning: Do not use Array index in keys  react/no-array-index-key
49:24  Warning: Do not use Array index in keys  react/no-array-index-key
60:30  Warning: Do not use Array index in keys  react/no-array-index-key
77:54  Warning: Do not use Array index in keys  react/no-array-index-key
78:26  Warning: Do not use Array index in keys  react/no-array-index-key
84:26  Warning: Do not use Array index in keys  react/no-array-index-key
95:25  Warning: Do not use Array index in keys  react/no-array-index-key
96:24  Warning: Do not use Array index in keys  react/no-array-index-key
103:28  Warning: Do not use Array index in keys  react/no-array-index-key

info  - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/basic-features/eslint#disabling-rules
> [PWA] Compile client (static)
> [PWA] Custom worker found: /home/vagrant/labelflow/typescript/web/src/worker/index.ts
> [PWA] Build custom worker: /home/vagrant/labelflow/typescript/web/public/worker-1OtgRMq_nv0oXA8O_At3B.js
> [PWA] Auto register service worker is disabled, please call following code in componentDidMount callback or useEffect hook
> [PWA]   window.workbox.register()
> [PWA] Service worker: /home/vagrant/labelflow/typescript/web/public/sw.js
> [PWA]   url: /sw.js
> [PWA]   scope: /
> [PWA] Fallback to precache routes when fetch failed from cache or network:
> [PWA]   document (page): /_offline
> [PWA] Inject manifest in /home/vagrant/labelflow/typescript/web/src/worker/index.ts
> [PWA] Compile server
[BABEL] Note: The code generator has deoptimised the styling of /home/vagrant/labelflow/node_modules/@apollographql/graphql-upload-8-fork/node_modules/busboy/deps/encoding/encoding-indexes.js as it exceeds the max of 500KB.
Webpack Bundle Analyzer saved report to /home/vagrant/labelflow/typescript/web/.next/static/bundle-analyzer/client.html
info  - Creating an optimized production build  
Failed to compile.

Sentry CLI Plugin: Command failed: /home/vagrant/labelflow/node_modules/@sentry/cli/sentry-cli releases new 1OtgRMq_nv0oXA8O_At3B
  DEBUG   2021-09-23 13:54:23.361064620 +00:00 sentry-cli version: 1.68.0, platform: "linux", architecture: "x86_64"
  INFO    2021-09-23 13:54:23.361157278 +00:00 sentry-cli was invoked with the following command line: "/home/vagrant/labelflow/node_modules/@sentry/cli/sentry-cli" "releases" "new" "1OtgRMq_nv0oXA8O_At3B"
  DEBUG   2021-09-23 13:54:23.361530927 +00:00 request POST https://sentry.io/api/0/projects/labelflow/labelflow/releases/
  DEBUG   2021-09-23 13:54:23.361564284 +00:00 json body: {"version":"1OtgRMq_nv0oXA8O_At3B","projects":["labelflow"],"dateStarted":"2021-09-23T13:54:23.361512306Z"}
  DEBUG   2021-09-23 13:54:23.361579694 +00:00 retry number 0, max retries: 0
  DEBUG   2021-09-23 13:54:23.880780537 +00:00 > POST /api/0/projects/labelflow/labelflow/releases/ HTTP/1.1
  DEBUG   2021-09-23 13:54:23.880808596 +00:00 > Host: sentry.io
  DEBUG   2021-09-23 13:54:23.880816313 +00:00 > Accept: */*
  DEBUG   2021-09-23 13:54:23.880822778 +00:00 > Connection: TE
  DEBUG   2021-09-23 13:54:23.880832383 +00:00 > TE: gzip
  DEBUG   2021-09-23 13:54:23.880839184 +00:00 > User-Agent: sentry-cli/1.68.0
  DEBUG   2021-09-23 13:54:23.880845417 +00:00 > Content-Type: application/json
  DEBUG   2021-09-23 13:54:23.880851570 +00:00 > Content-Length: 107
  DEBUG   2021-09-23 13:54:24.034431718 +00:00 < HTTP/1.1 401 Unauthorized
  DEBUG   2021-09-23 13:54:24.034481155 +00:00 < Server: nginx
  DEBUG   2021-09-23 13:54:24.034498248 +00:00 < Date: Thu, 23 Sep 2021 13:54:24 GMT
  DEBUG   2021-09-23 13:54:24.034511322 +00:00 < Content-Type: application/json
  DEBUG   2021-09-23 13:54:24.034523283 +00:00 < Content-Length: 58
  DEBUG   2021-09-23 13:54:24.034535337 +00:00 < Connection: keep-alive
  DEBUG   2021-09-23 13:54:24.034549841 +00:00 < www-authenticate: xBasic realm="api"
  DEBUG   2021-09-23 13:54:24.034561278 +00:00 < allow: GET, POST, HEAD, OPTIONS
  DEBUG   2021-09-23 13:54:24.034572960 +00:00 < access-control-allow-methods: GET, POST, HEAD, OPTIONS
  DEBUG   2021-09-23 13:54:24.034610946 +00:00 < access-control-allow-headers: X-Sentry-Auth, X-Requested-With, Origin, Accept, Content-Type, Authentication, Authorization, Content-Encoding
  DEBUG   2021-09-23 13:54:24.034624297 +00:00 < access-control-expose-headers: X-Sentry-Error, Retry-After
  DEBUG   2021-09-23 13:54:24.034635900 +00:00 < access-control-allow-origin: *
  DEBUG   2021-09-23 13:54:24.034647299 +00:00 < vary: Accept-Language, Cookie
  DEBUG   2021-09-23 13:54:24.034658759 +00:00 < content-language: en
  DEBUG   2021-09-23 13:54:24.034670045 +00:00 < x-frame-options: deny
  DEBUG   2021-09-23 13:54:24.034681710 +00:00 < x-content-type-options: nosniff
  DEBUG   2021-09-23 13:54:24.034694513 +00:00 < x-xss-protection: 1; mode=block
  DEBUG   2021-09-23 13:54:24.034712581 +00:00 < x-envoy-attempt-count: 1
  DEBUG   2021-09-23 13:54:24.034724069 +00:00 < x-envoy-upstream-service-time: 23
  DEBUG   2021-09-23 13:54:24.034735298 +00:00 < x-served-by: getsentry-web-default-production-944cb8f8-66j6s
  DEBUG   2021-09-23 13:54:24.034746325 +00:00 < x-served-by: lb-7
  DEBUG   2021-09-23 13:54:24.034758160 +00:00 < Strict-Transport-Security: max-age=31536000; includeSubDomains; preload
  DEBUG   2021-09-23 13:54:24.034795761 +00:00 response status: 401
  DEBUG   2021-09-23 13:54:24.034811842 +00:00 body: {"detail":"Authentication credentials were not provided."}
error: API request failed
  caused by: sentry reported an error: Authentication credentials were not provided. (http status: 401)
  DEBUG   2021-09-23 13:54:24.035668469 +00:00 client close; no transport to shut down  (from sentry)
  DEBUG   2021-09-23 13:54:24.035733828 +00:00 skipping update nagger because session is not attended


> Build error occurred
Error: > Build failed because of webpack errors
    at /home/vagrant/labelflow/node_modules/next/dist/build/index.js:390:19
    at async Span.traceAsyncFn (/home/vagrant/labelflow/node_modules/next/dist/telemetry/trace/trace.js:60:20)
```

**Comment by nicOwlas** [comment]
Thanks for bringing this up @mihav. We are looking into it.

I could reproduce with `yarn v3.0.1` and `node v15.5.1 `on Mac

**Output log**
```
➜  labelflow git:(main) yarn build:web

Prisma schema loaded from schema.prisma

✔ Generated Prisma Client (2.30.3) to ./../../node_modules/@prisma/client in 473ms
You can now start using Prisma Client in your code. Reference: https://pris.ly/d/client

import { PrismaClient } from '@prisma/client'
const prisma = new PrismaClient()

info  - Loaded env from /Users/nicolas/git/labelflow/typescript/web/.env.local
info  - Using webpack 5. Reason: Enabled by default https://nextjs.org/docs/messages/webpack5
warn  - You have enabled experimental feature(s).
warn  - Experimental features are not covered by semver, and may cause unexpected or broken application behavior. Use them at your own risk.

info  - Checking validity of types
warn  - The Next.js plugin was not detected in your ESLint configuration. See https://nextjs.org/docs/basic-features/eslint#migrating-existing-config

./src/pages/api/auth/[...nextauth].ts
60:7  Warning: Unexpected console statement.  no-console
64:7  Warning: Unexpected console statement.  no-console
67:7  Warning: Unexpected console statement.  no-console

./src/components/class-selection-popover/__stories__/class-selection-popover.tsx
64:3  Warning: Unexpected alert.  no-alert
77:32  Warning: Unexpected console statement.  no-console
94:32  Warning: Unexpected console statement.  no-console

./src/components/datasets/__stories__/dataset-card.tsx
23:22  Warning: Unexpected console statement.  no-console
24:24  Warning: Unexpected console statement.  no-console

./src/components/labelling-tool/drawing-tool-bar/__stories__/drawing-tool.tsx
44:33  Warning: Unexpected console statement.  no-console
46:28  Warning: Unexpected console statement.  no-console

./src/components/labelling-tool/options-tool-bar/class-selection-menu/__stories__/class-selection-menu.tsx
71:3  Warning: Unexpected alert.  no-alert

./src/components/website/Navbar/NavContent.tsx
59:34  Warning: Do not use Array index in keys  react/no-array-index-key
61:34  Warning: Do not use Array index in keys  react/no-array-index-key
97:29  Warning: Do not use Array index in keys  react/no-array-index-key

./src/components/website/Navbar/Submenu.tsx
37:22  Warning: Do not use Array index in keys  react/no-array-index-key
75:34  Warning: Do not use Array index in keys  react/no-array-index-key

./src/components/website/Pricing/PricingCard.tsx
60:46  Warning: Do not use Array index in keys  react/no-array-index-key

./src/components/website/Pricing/PricingTable.tsx
47:54  Warning: Do not use Array index in keys  react/no-array-index-key
49:24  Warning: Do not use Array index in keys  react/no-array-index-key
60:30  Warning: Do not use Array index in keys  react/no-array-index-key
77:54  Warning: Do not use Array index in keys  react/no-array-index-key
78:26  Warning: Do not use Array index in keys  react/no-array-index-key
84:26  Warning: Do not use Array index in keys  react/no-array-index-key
95:25  Warning: Do not use Array index in keys  react/no-array-index-key
96:24  Warning: Do not use Array index in keys  react/no-array-index-key
103:28  Warning: Do not use Array index in keys  react/no-array-index-key

info  - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/basic-features/eslint#disabling-rules
> [PWA] Compile client (static)
> [PWA] Custom worker found: /Users/nicolas/git/labelflow/typescript/web/src/worker/index.ts
> [PWA] Build custom worker: /Users/nicolas/git/labelflow/typescript/web/public/worker--nsVpGERi5kW-UVDqzaFk.js
> [PWA] Auto register service worker is disabled, please call following code in componentDidMount callback or useEffect hook
> [PWA]   window.workbox.register()
> [PWA] Service worker: /Users/nicolas/git/labelflow/typescript/web/public/sw.js
> [PWA]   url: /sw.js
> [PWA]   scope: /
> [PWA] Fallback to precache routes when fetch failed from cache or network:
> [PWA]   document (page): /_offline
> [PWA] Inject manifest in /Users/nicolas/git/labelflow/typescript/web/src/worker/index.ts
> [PWA] Compile server
[BABEL] Note: The code generator has deoptimised the styling of /Users/nicolas/git/labelflow/node_modules/@apollographql/graphql-upload-8-fork/node_modules/busboy/deps/encoding/encoding-indexes.js as it exceeds the max of 500KB.
Webpack Bundle Analyzer saved report to /Users/nicolas/git/labelflow/typescript/web/.next/static/bundle-analyzer/client.html
info  - Creating an optimized production build
Failed to compile.

Sentry CLI Plugin: Command failed: /Users/nicolas/git/labelflow/node_modules/@sentry/cli/sentry-cli releases new -nsVpGERi5kW-UVDqzaFk
error: API request failed
  caused by: sentry reported an error: Authentication credentials were not provided. (http status: 401)

Add --log-level=[info|debug] or export SENTRY_LOG_LEVEL=[info|debug] to see more output.
Please attach the full debug log to all bug reports.


> Build error occurred
Error: > Build failed because of webpack errors
    at /Users/nicolas/git/labelflow/node_modules/next/dist/build/index.js:390:19
    at async Span.traceAsyncFn (/Users/nicolas/git/labelflow/node_modules/next/dist/telemetry/trace/trace.js:60:20)
```

**Comment by nacho-carnicero** [comment]
I pushed a fix for this in #466, could you guys @mihav @nicOwlas check out that branch and let me know if it fixes the issue?
Thanks!

**Comment by mihav** [comment]
Yes, this branch builds successfully, thanks.
